### PR TITLE
Adding in local full stack deployment changes.

### PR DESCRIPTION
**Pull request by THOR300**
# Description

This pull request is a bug fix for a failing local deployment of the frontend and backend. 
- Change the vespa url that the backend uses. Port `19071` is the config endpoint whereas `8080` is the content endpoint, see the documentation [here](https://docs.vespa.ai/en/vespa-quick-start.html). 
> The port 8080 is published to make the search and feed interfaces accessible from outside the docker container, 19071 is the config server endpoint. Only one docker container named vespa can run at a time so change the name if needed. See [Docker containers](https://docs.vespa.ai/en/operations-selfhosted/docker-containers.html) for more insights.
- Creates a local certificates directory with empty cert and key. The vespa code just expects us to be passing these in or an api key. 
- Adds documentation and some initial draft code for spinning up ones local environment to test the frontend and backend stack. This includes details of dumping and loading data from vespa. 

_Note: We're not actually passing in the vespa url from the `.env` into the docker-compose is there a reason for this, do we want to change this?_  

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [x] Skip auto-tagging
- [ ] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] GitHub workflow update
- [ ] Documentation update
- [ ] Refactor legacy code
- [ ] Dependency update

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] **DB_CLIENT DEPENDENCY IS ON THE LATEST VERSION**
- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: docs/local_full_stack_setup/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+# Navigator Full Stack Local Setup
```
Comment (katybaulch) [inline docs/local_full_stack_setup/README.md:1]: Oh hell yeah - this is AWESOME!!!!

File: docs/local_full_stack_setup/family_document_sample.jsonl
Comment (katybaulch) [inline docs/local_full_stack_setup/family_document_sample.jsonl:1]: Just checking the file extension on this is correct? `.jsonl` ?

Comment (THOR300) [reply docs/local_full_stack_setup/family_document_sample.jsonl:1]: Yep correct

**Review by katybaulch — APPROVED** [review]
amazing ⭐ ⭐

**Pull request merged**